### PR TITLE
feat: add tspan support

### PR DESCRIPTION
**Pull request by nathanielc**
Adds `tspan` support so control over `text` is complete.

### CODE REVIEW

File: src/node/element/mod.rs [hunk 1]
```diff
@@ -227,6 +227,9 @@ implement! {
     #[doc = "A [`text`](https://www.w3.org/TR/SVG/text.html#TextElement) element."]
     struct Text
 
+    #[doc = "A [`tspan`](https://www.w3.org/TR/SVG/text.html#TextElement) element."]
+    struct TSpan
```
Comment (IvanUkhov) [inline src/node/element/mod.rs]: Please put it under `Title`.

File: src/node/element/tag.rs [hunk 2]
```diff
@@ -156,6 +156,7 @@ implement! {
     SVG: "svg",
     Symbol: "symbol",
     Text: "text",
+    TSpan: "tspan",
```
Comment (IvanUkhov) [inline src/node/element/tag.rs]: Please put it under `Title`.

**Review by IvanUkhov — COMMENTED** [review]
Nice catch! Not sure how I missed it. Thank you!

**Comment by nathanielc** [comment]
Updated, it was an easy enough change. Thanks for the crate its been handy.

**Comment by IvanUkhov** [comment]
Thank you!

**Pull request merged**